apps/src/templates/teacherDashboard/RecentCourses.story.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Got it. Okay, for now I'm going to leave the RecentCourses component as is, and build a more generic one in a separate PR. Then, when the new general CollapsibleSection is functional, I'll switch over to using it.